near-sdk/src/store/free_list/iter.rs

```diff
@@ -89,6 +89,15 @@ where
     }
 }
 
+impl<'a, T> Clone for Iter<'a, T>
```

Comment:
Wouldn't `#[derive(Clone)]` be enough? Are we absolutely need a manual implementation?

Reply:
yes, it is enough, that would work and would be more idiomatic.
i just thought, since most of the implementations were done manually, maybe that could be manual to. but that's unnecessary code i guess
i would change it to `#[derive(clone)]`